src/gen/expressionUtils.ts

```diff
@@ -171,7 +174,7 @@ export function _not(exp: ts.Expression): ts.Expression {
 export function _access(exp: ts.Expression, key: string | number | ts.Expression): ts.Expression {
     if (typeof key === "string") {
         if (isInt(key)) return factory.createElementAccessExpression(exp, ts.factory.createNumericLiteral(key));
-        return ts.factory.createPropertyAccessExpression(exp, key);
+        return ts.factory.createElementAccessExpression(exp, ts.factory.createStringLiteral(key));
```

Comment:
Wouldn't this always create element access expressions? It would be nice to create element access expressions only when necessary because otherwise code turns to this:

```ts
// We don't want this
errs["push"]("Expected value to be an object");
```

Reply:
It sure looks good to use createElementAccessExpression only when there are special characters, but since it is only used to access data and is a built file, it doesn't seem to matter much.